### PR TITLE
Implement

### PR DESCRIPTION
**Pull request by tomrf1**
A lambda for copying new files from an s3 bucket to an ftp server.
It takes as input an s3 event and streams objects to the configured ftp server.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,4 @@
+dist
+target
+node_modules
+.idea/
```
Comment (regiskuckaertz) [inline .gitignore:4]: tip: I suggest you add this line to your [global gitignore](https://help.github.com/articles/ignoring-files/), then you'll never need it again

File: src/local.ts [hunk 2]
```diff
@@ -0,0 +1,34 @@
+import { handler } from './lambda';
+let AWS = require('aws-sdk');
+
+/**
+ * For testing locally:
+ * `yarn run local <source bucket> <object key>`
```
Comment (regiskuckaertz) [inline src/local.ts:6]: for the future: It would be nice if we had a mocking library so that tests can happen in memory

Comment (tomrf1) [reply src/local.ts:6]: good point. This looks promising https://www.npmjs.com/package/aws-sdk-mock

**Review by regiskuckaertz — APPROVED** [review]
It's also a joy to read 🎉  I suggest you ignore lock files, but otherwise looking great!

**Pull request merged**